hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTaskWithLegacy.java

```diff
@@ -210,7 +217,13 @@ private void processWithObjectStoreLayout(OmKeyInfo updatedKeyInfo,
                                             OMDBUpdateEvent.OMDBUpdateAction action,
                                             Map<Long, NSSummary> nsSummaryMap)
       throws IOException {
-    setParentBucketId(updatedKeyInfo);
+    try {
+      setParentBucketId(updatedKeyInfo);
+    } catch (IOException e) {
+      LOG.warn("Skipping NSSummary creation for key: {} due to missing parent bucket info. Exception: {}",
```

Comment:
Same warn message we are logging in `setParentBucketId` method also. `processWithObjectStoreLayout` is the caller of `setParentBucketId`, can we log different message which can mention what that caller is doing.